src/sushi-hooks/queries/useTokenBalance.ts

```diff
@@ -36,6 +36,10 @@ const useTokenBalance = (tokenAddress?: string, accountAddress?: string) => {
     }
   }
 
+  useEffect(() => {
+    setTargetAddress(account)
```

Comment:
I think we still need this condition here (see l20) when assigning the targetAddress. We need to make it backwards compatible for others using `useTokenBalance()` hook which explicitly provides an `accountAddress` param

```suggestion
    setTargetAddress(accountAddress ?? account)
```

Reply:
@schystz done resolving your feedback